### ISSUE TITLE
First case first?

### ISSUE THREAD

**Issue by maxxbw54** [secureai4code/adversarial-codegen#12]
Thanks for ur hard work. @TomasAndersonFang @TheFatBlue 

Some details remain unclear to me, like what exactly the results will be shown, how the models will be evaluated, etc.

I am thinking maybe it's better to hv our first case based on first model to stabilize our pipelient and also demonstration. Then that will be a matter of expansion afterwards.

**Comment by maxxbw54** [comment]
At this moment, I'd highly recommend to play with small model first and releasing results. Big model may hv ethic concern if we make it public directly (attackers may take advantage of it).

**Comment by maxxbw54** [comment]
What's more valuable in my mind it's to keep building and improving a universal high-quality benchmark. So that it will be used by industry in their internal testing environment before they release their model. @TomasAndersonFang @TheFatBlue

**Comment by TomasAndersonFang** [comment]
> At this moment, I'd highly recommend to play with small model first and releasing results. Big model may hv ethic concern if we make it public directly (attackers may take advantage of it).

I agree with you. For deepseek-33b, I just want to see whether there is a consistent result between small LLM and big LLM.

**Comment by TomasAndersonFang** [comment]
> Thanks for ur hard work. @TomasAndersonFang @TheFatBlue
> 
> Some details remain unclear to me, like what exactly the results will be shown, how the models will be evaluated, etc.
> 
> I am thinking maybe it's better to hv our first case based on first model to stabilize our pipelient and also demonstration. Then that will be a matter of expansion afterwards.

Now we can clone our repo and install it according to the README. We also provide the easiest way to run our project.

**Comment by TomasAndersonFang** [comment]
> What's more valuable in my mind it's to keep building and improving a universal high-quality benchmark. So that it will be used by industry in their internal testing environment before they release their model. @TomasAndersonFang @TheFatBlue

You are right, and that's what we're doing now!

**Comment by TomasAndersonFang** [comment]
此issue已迁移到新仓库: https://github.com/safeai4code/RobustEval-CLM/issues/32

**Comment by TomasAndersonFang** [comment]
This issue has been migrated to the new repository: https://github.com/safeai4code/RobustEval-CLM/issues/32
